### DISCUSSION TITLE
TV2 Norway

### DISCUSSION THREAD

**Discussion by Sonicfan9** [iptv-org/community#795] [Q&A]
### Channel Name

TV2 Norway

### Channel ID

TV2Direkte.no

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://ws31-hls-live.akamaized.net/out/u/1416253.mpd ?

**Comment by Sonicfan9** [comment]
When I used that like to watch it, it just downloaded me a file that is unusable. I tried to use the link on one of the iptv apps, but it says that it's not valid.

**Comment by Sonicfan9** [comment]
Nevermind I got the link to work on one of the iptv apps on my phone, but when I tried to play it, it says that it's not playing

**Comment by trancer1994** [comment]
The link appears to be for the TV2 Sporten (Sport) which at the moment isn't playing anything because there's no sport events. Or at least that's what I get on my end. How were you able to pull it from the site though? I can't find the place where you listen to it

**Comment by Sonicfan9** [comment]
I didn't meant tv2 sporten, I meant the regular tv2

**Comment by Sonicfan9** [comment]
But for that, I didn't pull it

**Comment by trancer1994** [comment]
That link you supplied here appears to be the one for TV 2 Sporten but I'll check again.

**Comment by BellezaEmporium** [comment]
> The link appears to be for the TV2 Sporten (Sport) which at the moment isn't playing anything because there's no sport events. Or at least that's what I get on my end. How were you able to pull it from the site though? I can't find the place where you listen to it

https://tv2.no/live , only live that is free to watch.

**Comment by trancer1994** [comment]
https://ws31-hls-live.akamaized.net/out/u/1416253.mpd Yeah I got the same link. It says TV 2 Sporten underneath the live player.

**Comment by Sonicfan9** [comment]
What about TV 2 Zebra, TV 2 Livsstil, and TV 2 Nyhetskanalen?

**Comment by BellezaEmporium** [comment]
> TV 2 Zebra

Paid-only screening

**Comment by Sonicfan9** [comment]
And for the regular TV 2? Is that also the same thing?

**Comment by BellezaEmporium** [comment]
> And for the regular TV 2? Is that also the same thing?

Everything merged into TV2 Play, and it's only with a subscription.

**Comment by Sonicfan9** [comment]
Oh okay then. Thanks for the help

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19789
